Installation fails because gprbuild raises constraint error

(1) I cloned the repo unsing `git clone --recursive ....`
(2) Then installed langkit using `pip install -e git+https://github.com/AdaCore/langkit.git@v21.0.0#egg=langkit` (not doing so generates a warning `WARNING: Cannot look at git URL git+https://github.com/AdaCore/langkit.git@v21.0.0#egg=langkit because it does not support lookup as web pages.`
(3) installed `patchelf` (I think this could be added to the installation documentation, since it is not obvious until reading `setup.py`)
(4) Ran `pip install ".[devel]"`
I have no clue why this happens or if i am doing something wrong.

```
(venv) [rs@fedora RecordFlux-language]$ pip install ".[devel]"
Processing /home/rs/Development/PYTHON/PycharmProjects/RecordFlux-language
    ERROR: Command errored out with exit status 1:
     command: /home/rs/Development/PYTHON/PycharmProjects/RecordFlux/venv/bin/python3.9 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-6yqx8_zg/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-6yqx8_zg/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-s41w40x8
         cwd: /tmp/pip-req-build-6yqx8_zg/
    Complete output (111 lines):
    running egg_info
    creating /tmp/pip-pip-egg-info-s41w40x8/RecordFlux_language.egg-info
    writing /tmp/pip-pip-egg-info-s41w40x8/RecordFlux_language.egg-info/PKG-INFO
    writing dependency_links to /tmp/pip-pip-egg-info-s41w40x8/RecordFlux_language.egg-info/dependency_links.txt
    writing requirements to /tmp/pip-pip-egg-info-s41w40x8/RecordFlux_language.egg-info/requires.txt
    writing top-level names to /tmp/pip-pip-egg-info-s41w40x8/RecordFlux_language.egg-info/top_level.txt
    writing manifest file '/tmp/pip-pip-egg-info-s41w40x8/RecordFlux_language.egg-info/SOURCES.txt'
    Generating source for librflxlang...
    warning: The following parsing rules are not used: unqualified_variable
    Pretty-printing sources for librflxlang...
    Units remaining: 1      
    Generation complete!
    Building the generated source code
    Setup
       [mkdir]        object directory for project Langkit_Support
       [mkdir]        library directory for project Langkit_Support
       [mkdir]        object directory for project Librflxlang
       [mkdir]        library directory for project Librflxlang
    Compile
       [Ada]          langkit_support-vectors.adb
       [Ada]          langkit_support-adalog-abstract_relation.adb
       [Ada]          langkit_support-tree_traversal_iterator.adb
       [Ada]          langkit_support-adalog.ads
       [Ada]          langkit_support-packrat.adb
       [Ada]          langkit_support-bump_ptr-vectors.adb
       [Ada]          langkit_support-adalog-predicates.adb
       [Ada]          langkit_support-adalog-relations.adb
       [Ada]          langkit_support-array_utils.adb
       [Ada]          langkit_support-hashes.adb
       [Ada]          langkit_support.ads
       [Ada]          langkit_support-symbols.adb
       [Ada]          langkit_support-adalog-main_support.adb
       [Ada]          langkit_support-types.ads
       [Ada]          langkit_support-boxes.adb
       [Ada]          langkit_support-diagnostics.adb
       [Ada]          langkit_support-adalog-unify_one_side.adb
       [Ada]          langkit_support-adalog-unify.adb
       [Ada]          langkit_support-adalog-debug.adb
       [Ada]          langkit_support-adalog-logic_var.ads
       [Ada]          langkit_support-adalog-logic_ref.adb
       [Ada]          langkit_support-adalog-unify_lr.adb
       [Ada]          langkit_support-relative_get.adb
       [Ada]          langkit_support-text.adb
       [Ada]          langkit_support-adalog-eq_same.adb
       [Ada]          langkit_support-cheap_sets.adb
       [Ada]          langkit_support-symbols-precomputed.adb
       [Ada]          langkit_support-adalog-pure_relations.adb
       [Ada]          langkit_support-slocs.adb
       [Ada]          langkit_support-iterators.adb
       [Ada]          langkit_support-lexical_env.adb
       [Ada]          langkit_support-token_data_handlers.adb
       [Ada]          langkit_support-errors.ads
       [Ada]          langkit_support-diagnostics-output.adb
       [Ada]          langkit_support-images.adb
       [Ada]          langkit_support-bump_ptr.adb
       [Ada]          langkit_support-adalog-operations.adb
       [C]            librflxlang-gdb.c
       [Ada]          librflxlang-analysis.adb
       [Ada]          librflxlang-c.adb
       [Ada]          librflxlang-common.adb
       [Ada]          librflxlang-debug.adb
       [Ada]          librflxlang-implementation-c.adb
       [Ada]          librflxlang-implementation.adb
       [Ada]          librflxlang-introspection.adb
       [Ada]          librflxlang-introspection_implementation.adb
       [Ada]          librflxlang-iterators.adb
       [Ada]          librflxlang-lexer.adb
       [Ada]          librflxlang-lexer_implementation.adb
       [Ada]          librflxlang-lexer_state_machine.adb
       [Ada]          librflxlang-parsers.adb
       [Ada]          librflxlang-private_converters.ads
       [Ada]          librflxlang-public_converters.adb
       [Ada]          librflxlang.ads
    Build Libraries
       [gprlib]       gnatcoll_gmp.lexch
       [gprlib]       gnatcoll_iconv.lexch
       [link library] libgnatcoll_iconv.so
       [link library] libgnatcoll_gmp.so
    Using built-in specs.
    COLLECT_GCC=/home/rs/Development/GNAT_ADA/GNAT2020/bin/gcc
    COLLECT_LTO_WRAPPER=/home/rs/Development/GNAT_ADA/GNAT2020/bin/../libexec/gcc/x86_64-pc-linux-gnu/9.3.1/lto-wrapper
    OFFLOAD_TARGET_NAMES=nvptx-none
    Target: x86_64-pc-linux-gnu
    Configured with: ../src/configure --enable-languages=ada,c,c++ --enable-checking=release --enable-dual-exceptions --enable-offload-targets=nvptx-none --enable-_cxa_atexit --enable-threads=posix --with-bugurl=URL:mailto:report@adacore.com --disable-nls --without-libiconv-prefix --disable-libstdcxx-pch --disable-libada --disable-multilib --with-mpfr=/it/sbx/a2c2-2020/x86_64-linux/mpfr_stable-c/install --with-gmp=/it/sbx/a2c2-2020/x86_64-linux/gmp_stable-c/install --with-mpc=/it/sbx/a2c2-2020/x86_64-linux/mpc_stable-c/install --with-build-time-tools=/it/sbx/a2c2-2020/x86_64-linux/gcc-c/build/buildtools/bin --prefix=/it/sbx/a2c2-2020/x86_64-linux/gcc-c/pkg --build=x86_64-pc-linux-gnu
    Thread model: posix
    gcc version 9.3.1 20200430 (for GNAT Community 2020 20200818) (GCC)
    COMPILER_PATH=/home/rs/Development/GNAT_ADA/GNAT2020/bin/../libexec/gcc/x86_64-pc-linux-gnu/9.3.1/:/home/rs/Development/GNAT_ADA/GNAT2020/bin/../libexec/gcc/
    LIBRARY_PATH=/home/rs/Development/GNAT_ADA/GNAT2020/bin/../lib/gcc/x86_64-pc-linux-gnu/9.3.1/:/home/rs/Development/GNAT_ADA/GNAT2020/bin/../lib/gcc/:/home/rs/Development/GNAT_ADA/GNAT2020/bin/../lib/gcc/x86_64-pc-linux-gnu/9.3.1/../../../../lib64/:/lib/../lib64/:/usr/lib/../lib64/:/home/rs/Development/GNAT_ADA/GNAT2020/bin/../lib/gcc/x86_64-pc-linux-gnu/9.3.1/../../../../x86_64-pc-linux-gnu/lib/:/usr/lib/x86_64-linux-gnu/:/home/rs/Development/GNAT_ADA/GNAT2020/bin/../lib/gcc/x86_64-pc-linux-gnu/9.3.1/../../../:/lib/:/usr/lib/
    COLLECT_GCC_OPTIONS='-shared' '-o' '/tmp/pip-req-build-6yqx8_zg/contrib/gnatcoll-bindings/iconv/lib/relocatable/libgnatcoll_iconv.so.0.0' '-L/home/rs/Development/GNAT_ADA/GNAT2020/lib/gcc/x86_64-pc-linux-gnu/9.3.1//adalib' '-L/home/rs/Development/GNAT_ADA/GNAT2020/lib/gnatcoll.relocatable/' '-L/home/rs/Development/GNAT_ADA/GNAT2020/lib/gpr/relocatable/gpr/' '-L/home/rs/Development/GNAT_ADA/GNAT2020/lib/xmlada/xmlada_schema.relocatable/' '-L/home/rs/Development/GNAT_ADA/GNAT2020/lib/xmlada/xmlada_dom.relocatable/' '-L/home/rs/Development/GNAT_ADA/GNAT2020/lib/xmlada/xmlada_sax.relocatable/' '-L/home/rs/Development/GNAT_ADA/GNAT2020/lib/xmlada/xmlada_input.relocatable/' '-L/home/rs/Development/GNAT_ADA/GNAT2020/lib/xmlada/xmlada_unicode.relocatable/' '-v' '-mtune=generic' '-march=x86-64' '-dumpdir' '/tmp/pip-req-build-6yqx8_zg/contrib/gnatcoll-bindings/iconv/lib/relocatable/libgnatcoll_iconv.so.0.0.'
     /home/rs/Development/GNAT_ADA/GNAT2020/bin/../libexec/gcc/x86_64-pc-linux-gnu/9.3.1/collect2 -plugin /home/rs/Development/GNAT_ADA/GNAT2020/bin/../libexec/gcc/x86_64-pc-linux-gnu/9.3.1/liblto_plugin.so -plugin-opt=/home/rs/Development/GNAT_ADA/GNAT2020/bin/../libexec/gcc/x86_64-pc-linux-gnu/9.3.1/lto-wrapper -plugin-opt=-fresolution=/tmp/ccjrQPRL.res -plugin-opt=-pass-through=-lgcc -plugin-opt=-pass-through=-lgcc_s -plugin-opt=-pass-through=-lc -plugin-opt=-pass-through=-lgcc -plugin-opt=-pass-through=-lgcc_s --eh-frame-hdr -m elf_x86_64 -shared -o /tmp/pip-req-build-6yqx8_zg/contrib/gnatcoll-bindings/iconv/lib/relocatable/libgnatcoll_iconv.so.0.0 /lib/../lib64/crti.o /home/rs/Development/GNAT_ADA/GNAT2020/bin/../lib/gcc/x86_64-pc-linux-gnu/9.3.1/crtbeginS.o -L/home/rs/Development/GNAT_ADA/GNAT2020/lib/gcc/x86_64-pc-linux-gnu/9.3.1//adalib -L/home/rs/Development/GNAT_ADA/GNAT2020/lib/gnatcoll.relocatable/ -L/home/rs/Development/GNAT_ADA/GNAT2020/lib/gpr/relocatable/gpr/ -L/home/rs/Development/GNAT_ADA/GNAT2020/lib/xmlada/xmlada_schema.relocatable/ -L/home/rs/Development/GNAT_ADA/GNAT2020/lib/xmlada/xmlada_dom.relocatable/ -L/home/rs/Development/GNAT_ADA/GNAT2020/lib/xmlada/xmlada_sax.relocatable/ -L/home/rs/Development/GNAT_ADA/GNAT2020/lib/xmlada/xmlada_input.relocatable/ -L/home/rs/Development/GNAT_ADA/GNAT2020/lib/xmlada/xmlada_unicode.relocatable/ -L/home/rs/Development/GNAT_ADA/GNAT2020/bin/../lib/gcc/x86_64-pc-linux-gnu/9.3.1 -L/home/rs/Development/GNAT_ADA/GNAT2020/bin/../lib/gcc -L/home/rs/Development/GNAT_ADA/GNAT2020/bin/../lib/gcc/x86_64-pc-linux-gnu/9.3.1/../../../../lib64 -L/lib/../lib64 -L/usr/lib/../lib64 -L/home/rs/Development/GNAT_ADA/GNAT2020/bin/../lib/gcc/x86_64-pc-linux-gnu/9.3.1/../../../../x86_64-pc-linux-gnu/lib -L/usr/lib/x86_64-linux-gnu -L/home/rs/Development/GNAT_ADA/GNAT2020/bin/../lib/gcc/x86_64-pc-linux-gnu/9.3.1/../../.. -z origin -rpath /home/rs/Development/GNAT_ADA/GNAT2020/lib/gnatcoll.relocatable:/home/rs/Development/GNAT_ADA/GNAT2020/lib/gpr/relocatable/gpr:/home/rs/Development/GNAT_ADA/GNAT2020/lib/xmlada/xmlada_schema.relocatable:/home/rs/Development/GNAT_ADA/GNAT2020/lib/xmlada/xmlada_dom.relocatable:/home/rs/Development/GNAT_ADA/GNAT2020/lib/xmlada/xmlada_sax.relocatable:/home/rs/Development/GNAT_ADA/GNAT2020/lib/xmlada/xmlada_input.relocatable:/home/rs/Development/GNAT_ADA/GNAT2020/lib/xmlada/xmlada_unicode.relocatable:/home/rs/Development/GNAT_ADA/GNAT2020/lib/gcc/x86_64-pc-linux-gnu/9.3.1/adalib:/home/rs/Development/GNAT_ADA/GNAT2020/lib64: -soname libgnatcoll_iconv.so.0 /tmp/pip-req-build-6yqx8_zg/contrib/gnatcoll-bindings/iconv/obj/relocatable/gnatcoll-iconv.o /tmp/pip-req-build-6yqx8_zg/contrib/gnatcoll-bindings/iconv/obj/relocatable/iconv_support.o -lgnatcoll -lgpr -lxmlada_schema -lxmlada_dom -lxmlada_sax -lxmlada_input_sources -lxmlada_unicode -lgnat-2020 -lgcc --push-state --as-needed -lgcc_s --pop-state -lc -lgcc --push-state --as-needed -lgcc_s --pop-state /home/rs/Development/GNAT_ADA/GNAT2020/bin/../lib/gcc/x86_64-pc-linux-gnu/9.3.1/crtendS.o /lib/../lib64/crtn.o
       [gprlib]       langkit_support.lexch
    /home/rs/Development/GNAT_ADA/GNAT2020/bin/../libexec/gcc/x86_64-pc-linux-gnu/9.3.1/ld: /tmp/pip-req-build-6yqx8_zg/contrib/gnatcoll-bindings/iconv/obj/relocatable/iconv_support.o: relocation R_X86_64_32 against `.rodata.str1.1' can not be used when making a shared object; recompile with -fPIC
    collect2: error: ld returned 1 exit status
    gprlib: /home/rs/Development/GNAT_ADA/GNAT2020/bin/gcc execution error
    gprbuild: raised CONSTRAINT_ERROR : gprbuild-post_compile.adb:1113:39 range check failed
    value 0 not in 1..2147483647
    [/home/rs/Development/GNAT_ADA/GNAT2020/bin/gprbuild]
    0x5261f6
    0x537df6
    0x53e3db
    0x5478aa
    0x5488b0
    0x4b465f
    0x4a4474
    [/lib64/libc.so.6]
    0x7fba7b9371e0
    [/home/rs/Development/GNAT_ADA/GNAT2020/bin/gprbuild]
    0x4a44c7
    
    Build failed: error while running gprbuild -p -j8 -P/tmp/pip-req-build-6yqx8_zg/lib/gnat/librflxlang.gpr -XBUILD_MODE=dev -XLIBRARY_TYPE=relocatable -XGPR_BUILD=relocatable -XXMLADA_BUILD=relocatable -aP /tmp/pip-req-build-6yqx8_zg/contrib/gnatcoll-bindings/iconv -aP /tmp/pip-req-build-6yqx8_zg/contrib/gnatcoll-bindings/gmp:
        Command '['gprbuild', '-p', '-j8', '-P/tmp/pip-req-build-6yqx8_zg/lib/gnat/librflxlang.gpr', '-XBUILD_MODE=dev', '-XLIBRARY_TYPE=relocatable', '-XGPR_BUILD=relocatable', '-XXMLADA_BUILD=relocatable', '-aP', '/tmp/pip-req-build-6yqx8_zg/contrib/gnatcoll-bindings/iconv', '-aP', '/tmp/pip-req-build-6yqx8_zg/contrib/gnatcoll-bindings/gmp']' returned non-zero exit status 4.
```

> Is that error reproducible when you use a clean workspace (a new venv and a clean clone of the repository) and just do steps (1) and (4)?

Yes, I tried it several times with a clean clone and clean venv, different python versions and different versions of `patchelf` 
(0.10 is the default on Debian and 0.12 is the latest version)

Now i tried to build it in a clean Debian 10 VM with Python 3.8.8 and the newest available GNAT compiler (20200429-93). Pip, setuptools are also the latest versions. `patchelf` has version 0.10. Now I am getting a different error and strangely some new directories are created 
```
[mkdir]        object directory for project GnatColl_GMP
 [mkdir]        library directory for project GnatColl_GMP
 [mkdir]        object directory for project GnatColl_Iconv
 [mkdir]        library directory for project GnatColl_Iconv:
```

```
$ pip install ".[develop]"
Processing /home/deb/RecordFlux-language
    ERROR: Command errored out with exit status 1:
     command: /home/deb/RecordFlux-language/venv/bin/python3.8 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-rdt4xdhs/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-rdt4xdhs/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-1xjh79yk
         cwd: /tmp/pip-req-build-rdt4xdhs/
    Complete output (97 lines):
    WARNING: Cannot look at git URL git+https://github.com/AdaCore/langkit.git@v21.0.0#egg=langkit because it does not support lookup as web pages.
    running egg_info
    creating /tmp/pip-pip-egg-info-1xjh79yk/RecordFlux_language.egg-info
    writing /tmp/pip-pip-egg-info-1xjh79yk/RecordFlux_language.egg-info/PKG-INFO
    writing dependency_links to /tmp/pip-pip-egg-info-1xjh79yk/RecordFlux_language.egg-info/dependency_links.txt
    writing requirements to /tmp/pip-pip-egg-info-1xjh79yk/RecordFlux_language.egg-info/requires.txt
    writing top-level names to /tmp/pip-pip-egg-info-1xjh79yk/RecordFlux_language.egg-info/top_level.txt
    writing manifest file '/tmp/pip-pip-egg-info-1xjh79yk/RecordFlux_language.egg-info/SOURCES.txt'
    Generating source for librflxlang...
    warning: The following parsing rules are not used: unqualified_variable
    warning: Black not available, not pretty-printing Python code
    warning: Black not available, not pretty-printing Python code
    warning: Black not available, not pretty-printing Python code
    warning: Black not available, not pretty-printing Python code
    Pretty-printing sources for librflxlang...
    Units remaining: 1      
    Generation complete!
    Building the generated source code
    Setup
       [mkdir]        object directory for project GnatColl_GMP
       [mkdir]        library directory for project GnatColl_GMP
       [mkdir]        object directory for project GnatColl_Iconv
       [mkdir]        library directory for project GnatColl_Iconv
       [mkdir]        object directory for project Langkit_Support
       [mkdir]        library directory for project Langkit_Support
       [mkdir]        object directory for project Librflxlang
       [mkdir]        library directory for project Librflxlang
    Compile
       [Ada]          langkit_support-symbols.adb
       [Ada]          langkit_support-adalog-unify.adb
       [Ada]          langkit_support-adalog-main_support.adb
       [Ada]          langkit_support-packrat.adb
       [Ada]          langkit_support-boxes.adb
       [Ada]          langkit_support-bump_ptr-vectors.adb
       [Ada]          langkit_support-types.ads
       [Ada]          langkit_support-adalog-abstract_relation.adb
       [Ada]          langkit_support-adalog-relations.adb
       [Ada]          langkit_support-diagnostics.adb
       [Ada]          langkit_support-adalog-pure_relations.adb
       [Ada]          langkit_support-adalog-operations.adb
       [Ada]          langkit_support-array_utils.adb
       [Ada]          langkit_support-token_data_handlers.adb
       [Ada]          langkit_support-cheap_sets.adb
       [Ada]          langkit_support.ads
       [Ada]          langkit_support-slocs.adb
       [Ada]          langkit_support-iterators.adb
       [Ada]          langkit_support-adalog-eq_same.adb
       [Ada]          langkit_support-adalog-predicates.adb
       [Ada]          langkit_support-lexical_env.adb
       [Ada]          langkit_support-images.adb
       [Ada]          langkit_support-errors.ads
       [Ada]          langkit_support-vectors.adb
       [Ada]          langkit_support-adalog-logic_var.ads
       [Ada]          langkit_support-hashes.adb
       [Ada]          langkit_support-adalog-unify_lr.adb
       [Ada]          langkit_support-diagnostics-output.adb
       [Ada]          langkit_support-bump_ptr.adb
       [Ada]          langkit_support-relative_get.adb
       [Ada]          langkit_support-adalog-logic_ref.adb
       [Ada]          langkit_support-text.adb
       [Ada]          langkit_support-symbols-precomputed.adb
       [Ada]          langkit_support-adalog-unify_one_side.adb
       [Ada]          langkit_support-adalog-debug.adb
       [Ada]          langkit_support-adalog.ads
       [Ada]          langkit_support-tree_traversal_iterator.adb
       [C]            iconv_support.c
       [Ada]          gnatcoll-iconv.adb
       [C]            gnatcoll-gmp-mpz_odd_p.c
    /tmp/pip-req-build-rdt4xdhs/contrib/gnatcoll-bindings/gmp/gnatcoll-gmp-mpz_odd_p.c:20:10: fatal error: gmp.h: No such file or directory
     #include <gmp.h>
              ^~~~~~~
    compilation terminated.
 
       compilation of gnatcoll-gmp-mpz_odd_p.c failed
 
    gprbuild: *** compilation phase failed
```

--- a/README.md
+++ b/README.md
@@ -4,7 +4,15 @@ This is the parser for the [RecordFlux](https://github.com/Componolit/RecordFlux
 
 ## Development
 
-As a prerequisite to build the parser, [GNAT Community Edition](https://www.adacore.com/download) >= 2020 needs to be installed. To run the tests, the RecordFlux language package and its dependencies must also be installed. Either use the respective make target:
+As a prerequisite to build the parser, the following dependencies need to be installed:
+
+- [GNAT Community Edition](https://www.adacore.com/download) >= 2020
+- patchelf
+- libgmp-dev (Debian) or gmp-devel (Fedora)
+
+Add `<GNAT-directory>/bin` to the PATH environment variable. It is recommended to prepend it to the existing $PATH.
+
+To run the tests, the RecordFlux language package and its dependencies must also be installed. Either use the respective make target:
 
 ```Console
 $ make install_parser

--- a/README.md
+++ b/README.md
@@ -7,10 +7,11 @@ This is the parser for the [RecordFlux](https://github.com/Componolit/RecordFlux
 As a prerequisite to build the parser, the following dependencies need to be installed:
 
 - [GNAT Community Edition](https://www.adacore.com/download) >= 2020
-- patchelf
-- libgmp-dev (Debian) or gmp-devel (Fedora)
+- [patchelf](https://github.com/NixOS/patchelf)
+- [GNU MP Arithmetic Library](https://gmplib.org/) + development tools. These are provided as packages for various distributions e.g. libgmp-dev (Debian/Ubuntu), gmp-devel (Fedora) or gmp (Arch)
 
-Add `<GNAT-directory>/bin` to the PATH environment variable. It is recommended to prepend it to the existing $PATH.
+**Note**: GNAT must be added to the PATH environment variable after installation: `export PATH=<GNAT-directory>/bin:$PATH`.
+
 
 To run the tests, the RecordFlux language package and its dependencies must also be installed. Either use the respective make target:
 

--- a/README.md
+++ b/README.md
@@ -8,7 +8,7 @@ As a prerequisite to build the parser, the following dependencies need to be ins
 
 - [GNAT Community Edition](https://www.adacore.com/download) >= 2020
 - [patchelf](https://github.com/NixOS/patchelf)
-- [GNU MP Arithmetic Library](https://gmplib.org/) + development tools. These are provided as packages for various distributions e.g. libgmp-dev (Debian/Ubuntu), gmp-devel (Fedora) or gmp (Arch)
+- [GNU MP Arithmetic Library](https://gmplib.org/) This is provided as a package for various distributions e.g. libgmp-dev (Debian/Ubuntu), gmp-devel (Fedora) or gmp (Arch Linux).
 
 **Note**: GNAT must be added to the PATH environment variable after installation: `export PATH=<GNAT-directory>/bin:$PATH`.
 

